packages/govuk-frontend/src/govuk/common/configuration/extract-config-by-namespace.jsdom.test.mjs

```diff
@@ -42,13 +42,17 @@ describe('extractConfigByNamespace', () => {
   it('defaults to empty config for known namespaces only', () => {
     const { dataset } = $element
 
-    const nonObject1 = extractConfigByNamespace(Component, dataset, 'a')
-    const nonObject2 = extractConfigByNamespace(Component, dataset, 'd')
-    const nonObject3 = extractConfigByNamespace(Component, dataset, 'e')
+    const nonObject1 = extractConfigByNamespace(Component.schema, dataset, 'a')
```

Comment:
**suggestion** Seems we're only using the `Component` to access the `.schema` in this test file. That means we can drop the class in favour of just storing the `schema` in a variable to simplify the tests a little bit (I think we'll still need to annotate that variable with `@satisfies`).